README.md

```diff
@@ -32,28 +32,30 @@ Winit is designed to be a low-level brick in a hierarchy of libraries. Consequen
 show something on the window you need to use the platform-specific getters provided by winit, or
 another library.
 
-```rust
+```rust, ignore
```

Comment:
By adding `ignore` you disable the checks on the CI, so it's pointless to test with `C` trick in `lib.rs`. Remove the `ignore` and address the CI errors.

Reply:
I think that's partly my fault; it should've been `no_run`